node/pkg/driver/node_utils.go

```diff
@@ -239,6 +243,25 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
+func (n NodeUtils) IsFCExists() bool {
+	if n.IsPathExists(FCPath) {
+		return n.IsDirHasContent(FCPath)
+	}
+	return false
+}
+
+func (n NodeUtils) IsDirHasContent(path string) bool {
+	f, _ := os.Open(path)
+	defer f.Close()
+
+	_, err := f.Readdir(1)
+
+	if err == io.EOF {
+		return false
+	}
+	return true
+}
```

Comment:
We can simplify this by:
```suggestion
	return err != io.EOF
```

Reply:
No other errors are relevant here?

> If n > 0, Readdir returns at most n FileInfo structures. In this case, if Readdir returns an empty slice, it will return a non-nil error explaining why. At the end of a directory, the error is io.EOF.